doc/python/decor.rst

```diff
@@ -71,17 +71,19 @@ Cable cell decoration
 
     .. method:: paint(region, Vm=None, cm=None, rL=None, tempK=None)
 
-        Set cable properties on a region.
+        Set cable properties on a region. If a pair of float ``v`` and a string
```

Comment:
This should probably be changed to "float or string" as well. Same with the following lines.